docs/architecture/addr_announcer.md

```diff
@@ -1,44 +0,0 @@
-# AddrAnnouncer
```

Comment:
Now that this is removed, can we get a description of how this works after the PR?